QuestionBot/QuestionBot.UnitTests/Model/QuestionMessageListenerTests.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System;
+using System.Linq;
+using NUnit.Framework;
+using System.Collections.Generic;
+using QuestionBot.Model;
+
+namespace QuestionBot.UnitTests.Model{
+
+    [TestFixture]
+    class QuestionMessageListenerTests{
+        private IMessageListener _testListener;
+        private IStore _storeTest;
+
+        [SetUp]
+        public void Setup(){
+            _storeTest = new InMemoryStore();
+            _testListener = new QuestionMessageListener(_storeTest);
```

Comment:
That makes sense, and I was leaving out certain tests because I knew the store would test those - and I shouldn't rely on that! Like you say, assume everything is trying to break your code